store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/FederatedAddGraphHandlerParent.java

```diff
@@ -93,6 +98,20 @@ public Void doOperation(final OP operation, final Context context, final Store s
         return null;
     }
 
+    private static void processNonViableFedStoreWithNonPersistentCaches(final GraphSerialisable graphSerialisable) throws OperationException {
+        final String storeClass = graphSerialisable.getStoreProperties().getStoreClass();
+        final Class<? extends Store> aClass;
+        try {
+            aClass = Class.forName(storeClass).asSubclass(Store.class);
```

Comment:
I think usually in Gaffer `clazz` is used but it's not a big deal

Reply:
it was autogenerated.